### PR TITLE
Update custom-connector-sdk-overview.md

### PR DESCRIPTION
**Pull request by rchanda1392**

### CODE REVIEW

**Review by Lauragra — APPROVED** [review]

**Comment by rchanda1392** [comment]
Hi @Lauragra I have updated the documentation to reflect that we are going GA and have made the corresponding changes. Please help with your review.

File: concepts/custom-connector-sdk-publish.md [hunk 1]
```diff
@@ -60,7 +39,7 @@ Use the following steps to publish a connection for your custom connector:
 
     The data in the parameter is opaque to the platform. It's serialized and stored as a string and passed to the connector. The connector can deserialize this data as required and use it.
 
-7. On the [Assign property labels](/microsoftsearch/configure-connector#step-6-assign-property-labels) page, assign semantic labels to your source properties.
+7. On the [Assign property labels](/microsoftsearch/configure-connector#step-6-assign-property-labels) page, assign semantic labels to your source properties. The default MRT will not work if semantic labels are not assigned.
```
Comment (Lauragra) [inline concepts/custom-connector-sdk-publish.md:42]: What does the acronym MRT stand for?

**Review by Lauragra — APPROVED** [review]

File: concepts/custom-connector-sdk-publish.md (hunk 1 above)
Comment (Lauragra) [reply concepts/custom-connector-sdk-publish.md:42]: Is this a commonly used/familiar acronym? Or can we spell it out as "modern result type"? Not sure that the term is a proper noun/should be capped?

Comment (rchanda1392) [reply concepts/custom-connector-sdk-publish.md:42]: We can just say that the default result type will not work....

**Pull request merged**